src/components/Buyer/CarbonCredits.js

```diff
@@ -1,7 +1,24 @@
-import React from "react";
+import React,{useState, useEffect} from "react";
+import CarbonCredit from "./CarbonCredit";
+import "./CarbonStyle.css"
 
 function CarbonCredits() {
-  return <div>CarbonCredits</div>;
+const [carbonCard, setCarbonCard]=useState([]);
+useEffect(() => {
+  fetch("/api/v1/carbon_credits")
+  .then((resp)=> resp.json())
+  .then((data)=>{
+    setCarbonCard(data);
+  });
+}, []);
+  return(
+    <> 
+    <p className="title">Available Carbon Projects</p>
+  <div className="gridcard"> 
+    <CarbonCredit content={carbonCard}/>
+  </div>
+  <button className="moreOpt">See more</button>
```

Comment:
-Replace this button with the following so we can have it in the middle as it is in the figma design 
`<div className="d-flex justify-content-center">
        <button className="btn btn-outline-success text-center">
          Buy Carbon Credits
        </button>
      </div>`